[2022-01-25 19:36:56,256] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.move manual__2022-01-25T19:36:54.952812+00:00 [queued]>
[2022-01-25 19:36:56,285] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.move manual__2022-01-25T19:36:54.952812+00:00 [queued]>
[2022-01-25 19:36:56,285] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 19:36:56,285] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-25 19:36:56,286] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 19:36:56,310] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): move> on 2022-01-25 19:36:54.952812+00:00
[2022-01-25 19:36:56,321] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'move', 'manual__2022-01-25T19:36:54.952812+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/dag_01.py', '--cfg-path', '/tmp/tmpnrjdt4co', '--error-file', '/tmp/tmpo7gjp1b8']
[2022-01-25 19:36:56,322] {standard_task_runner.py:77} INFO - Job 99: Subtask move
[2022-01-25 19:36:56,316] {standard_task_runner.py:52} INFO - Started process 3701 to run task
[2022-01-25 19:36:56,433] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.move manual__2022-01-25T19:36:54.952812+00:00 [running]> on host 223810fcea28
[2022-01-25 19:36:56,523] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=move
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T19:36:54.952812+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-25T19:36:54.952812+00:00
[2022-01-25 19:36:56,525] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 19:36:56,526] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/']
[2022-01-25 19:36:56,543] {subprocess.py:85} INFO - Output:
[2022-01-25 19:36:56,547] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 19:36:56,589] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=move, execution_date=20220125T193654, start_date=20220125T193656, end_date=20220125T193656
[2022-01-25 19:36:56,661] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 19:36:56,764] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
